tests/data/fixture_data.py

```diff
@@ -344,6 +306,7 @@
         FIELD_COG_BARCODE: "hij",
         FIELD_ROOT_SAMPLE_ID: "MCM010",
         FIELD_DATE_TESTED: DATE_TESTED_NOW,
+        FIELD_FILTERED_POSITIVE: True,
     },
 ]
 
```

Comment:
Is there a test for filtered positive = null? just in case it doesn't get set for whatever reason.

Reply:
🤔 good point, what are we expecting? Probably that that sample is deemed `filtered_positive = False`?